build_daemon/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
 
 dev_dependencies:
   build_runner: ^1.0.0
-  built_value_generator: ^7.0.0
+  built_value_generator: ^8.0.0
```

Comment:
I believe we actually need to keep this at the `^7.0.0` constraint - or we also need to bump the `built_value` constraint to `^8.0.0`.

Reply:
Done.  This turned out not to be needed.